compiler-cli/src/fs.rs

```diff
@@ -54,6 +54,34 @@ pub fn get_project_root(path: Utf8PathBuf) -> Result<Utf8PathBuf, Error> {
     })
 }
 
+#[inline]
+pub fn get_os() -> OS {
+    OS::from(std::env::consts::OS)
+}
+
+// Return the distro enum if /etc/os-release exists, otherwise return Other
+pub fn get_os_distro() -> Distro {
+    if let OS::Linux = get_os() {
```

Comment:
This function is a little hard to read and uses a lot of nesting. Could you rewrite to only use 1 level of indentation please by using early returns